dgraph/cmd/counter/increment.go

```diff
@@ -74,6 +75,15 @@ func queryCounter(txn *dgo.Txn, pred string) (Counter, error) {
 	var counter Counter
 	query := fmt.Sprintf("{ q(func: has(%s)) { uid, val: %s }}", pred, pred)
 	resp, err := txn.Query(ctx, query)
+
+	// Total query latency is sum of encoding, parsing and processing
+	// latencies
+	queryLatency := resp.Latency.GetEncodingNs() +
+		resp.Latency.GetParsingNs() + resp.Latency.GetProcessingNs()
+	latency := strconv.FormatUint(queryLatency, 10) + "ns"
```

Comment:
This seems unnecessary. You could just say
`latency := time.NanoSecond * queryLatency`